spk/rutorrent/src/dsm-control.sh

```diff
@@ -6,7 +6,7 @@ DNAME="ruTorrent"
 
 # Others
 INSTALL_DIR="/usr/local/${PACKAGE}"
-PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/usr/bin:${PATH}"
+export PATH="${INSTALL_DIR}/env/bin:${INSTALL_DIR}/bin:${INSTALL_DIR}/usr/bin:/bin:${PATH}"
```

Comment:
We add both python virtual environment directory (created only if python3 is here) and `/bin` directory (where `stat` can be taken from)